### ISSUE TITLE
Login Section Fields do not show up as planned

### ISSUE THREAD

**Issue by IP-CAM** [Feodor2/Mypal68#354]
Login Section Fields do not show up as planned on latest Version Mypal: 
![empty-fields](https://github.com/Feodor2/Mypal68/assets/12776547/8b5d59ab-e602-4100-b760-01b5f0c6ec89)

**Comment by Feodor2** [comment]
SSE version certainly has bug with graphics on newer machines, shuold take it only for p3 or athlon xp level machines

**Comment by IP-CAM** [comment]
I am using my old WIN XP SP-3 PC on this.

**Comment by Feodor2** [comment]
Okey, this is not the matter.
When it was happened, and did you took the new version 68.13.8

**Comment by IP-CAM** [comment]
> Okey, this is not the matter. When it was happened, and did you took the new version 68.13.8

Yes, I made the Test with the v.68.13.8

**Comment by Feodor2** [comment]
So its not suitable for your machine which is newer, return to previous version.

**Comment by Sen-nin** [comment]
Something similar happens to the login form of WHM (most common webhosting platform) as well. You pick the right user name for the user name field, and then both the user name and the password disappear. To check the password, you click on the make-it-visible box, but nothing happens.

Downgrading to the previous version requires a new profile, and it is troublesome. So I think people who do not use a Pentium 3 or Athlon XP computer should avoid the 68.13.8b.

**Comment by Feodor2** [comment]
Hey, give the link please, I want to fix this, I am going to publish the new sse version upon the weekend.

**Comment by IP-CAM** [comment]
With MyPal 68.14.2b it works again.

**Comment by Feodor2** [comment]
I asked this for to fix the sse version, well I publish it as is not fixed then.

**Comment by IP-CAM** [comment]
It was this Site, where MyPal did not show tzhe login Section::
https://forum.opencart.com/index.php

**Comment by Sen-nin** [comment]
I got this from Feodor2 today after I created a new issue thread:

> Is this not same as the https://github.com/Feodor2/Mypal68/issues/354? Why duplicate
> Again you do not give links, how I suppose to check, okey I can type outlook.com and other what, and I see nothing wrong now on outlook, sse or not see it is the same
> 

That is funny. It happened at outlook.com, and cPanel web mail. Actually in other places as well.

This did not happen with the nonsse version. When I deleted the nonsse version and switched to the sse version, this happened. I have gotten the nonsse version back, so I cannot show it to you anymore. I will upload the captures again so other people can see it.

Plus, login itself can be performed. Just what is in the text (input) area is invisible.

I don't know why Feodor2 cannot confirm it. Is that due to your browser cache or something? It is rather embarrassing.

Also sorry for the duplication. I forgot about this thread.

![login-problem-outlook-trimmed-and-erased](https://github.com/user-attachments/assets/7ee9449d-6729-414b-addb-96dc7f685bc0)

![Uploading roundcube-login-trimmed-erased.png…]()

![roundcube-login-credentials-entered-trimmed](https://github.com/user-attachments/assets/96f5ab43-2362-4fda-8533-c3a222d3e002)

**Comment by Sen-nin** [comment]
Another two captures of cPanel web mail.

![roundcube-login-trimmed-erased](https://github.com/user-attachments/assets/5b4bfa72-cdfd-412d-af76-bea6376d28be)

![roundcube-login-credentials-entered-trimmed](https://github.com/user-attachments/assets/0627e808-9460-4a73-aa7b-984cb3194273)

**Comment by Feodor2** [comment]
Yes I can see the text when type login on opencart and outlook and on the sse version, another one don't know because no was given the link.

**Comment by Sen-nin** [comment]
I do not understand what you mean, Feodor2. You can see what you have typed in the login form when you are on the SSE version?

Also, which one were you referring to when you said "no was given the link"?

**Comment by Feodor2** [comment]
Yes the text is shown normally and about the link is for webmail picture.

**Comment by Sen-nin** [comment]
I checked some Windows XP computers, and I was unable to find this problem occur on ThinkPad T43 No. 3 and No. 5 computers and ThinkPad X201i No. 6 computer. I do not remember on which computer I found this problem, but I will check out another one tonight (13 hours to 18 hours from now). Maybe there is some kind of conflict on the computer on which this problem happens. The same is true with the PayPal problem too. That problem did not happen with the above three computers.

**Comment by Sen-nin** [comment]
I checked my ThinkPad T43 No. 1 computer, and no problem was found with it. I guess I will have to check out T43 No. 2 computer then (I have not used any computers other than these lately). That will be 24 hours from now, but it is getting to look like some kind of conflict which happened only on a specific computer.

**Comment by Sen-nin** [comment]
I am sorry it took so long but I found no problem with other ThinPad T43 computers. It is funny but it may have been caused by some kind of cache or some kind of conflict since I certainly confirmed the problem before I reported it here. Please forget it. I will report it if something like this should happen again. I am sorry for the trouble I caused.

**Comment by Sen-nin** [comment]
I was wrong. The PayPal login issue does really happen. It happens when you put something in a shopping cart and pay for it by PayPal. You cannot fill in the provided field with the code PayPal sent you by SMS whatever way, manually type it or click on the up or down arrow in each field. This probably does not seem to happen when you just login to your PayPal account. I found it out when I tried to renew some of my domains at a registrar. In the end I had to do it with my tablet.
